vite/vite.w

```diff
@@ -0,0 +1,139 @@
+bring cloud;
+bring sim;
+bring util;
+bring fs;
+bring "./find-port.w" as find_port;
+
+struct SimViteProps {
+  root: str;
+  env: Map<str>;
```

Comment:
```suggestion
  env: Map<str>?;
```

Reply:
I made it required for the target implementations because there's some env variables we are forced to pass, otherwise things won't work (eg, `PATH` and `HOME`). Both targets needed them.

Reply:
in this case i would "elevate" them into explicit apis such as `path` and `home` and then plug them into the environment below.